com.unity.render-pipelines.lightweight/Editor/RendererFeatures/NewRendererFeatureDropdownItem.cs

```
@@ -0,0 +1,20 @@
+using UnityEngine.Rendering.LWRP;
+
+namespace UnityEditor.Rendering.LWRP
+{
+    public static class NewRendererFeatureDropdownItem
+    {
+        static readonly string defaultNewClassName = "CustomRenderPassFeature.cs";
+        static readonly string templateGuid = "51493ed8d97d3c24b94c6cffe834630b";
+
+        [MenuItem("Assets/Create/Rendering/Lightweight Render Pipeline/Renderer Feature", priority = EditorUtils.lwrpAssetCreateMenuPriorityGroup2)]
+        public static void CreateNewRendererFeature()
+        {
+            string templatePath = AssetDatabase.GUIDToAssetPath(templateGuid);
+            if (string.IsNullOrEmpty(templatePath))
+                templatePath = LightweightRenderPipelineAsset.packagePath + "/Editor/RendererFeatures/NewRendererFeature.cs.txt";
```

Is this fallback actually needed? Seems like it would just make it fail silently. I'd recommend adding a unit test that asserts `string.IsNullOrEmpty(AssetDatabase.GUIDToAssetPath(templateGuid))`.

---

Fixed. Added test.